app/Console/Kernel.php

```diff
@@ -32,48 +32,129 @@ protected function schedule(Schedule $schedule): void
         $schedule->command('update:cloudfront-ips')
             ->hourly();
 
-        $schedule->command('delete:collections')
-            ->dailyAt('23:00')
-            ->withoutOverlapping(self::FOR_ONE_YEAR)
-            ->sendOutputTo(storage_path('logs/delete-collections-last-run.log'));
-
+        //
+        // Mobile app
         $schedule->command('import:mobile')
             ->dailyAt('23:05')
             ->withoutOverlapping(self::FOR_ONE_YEAR)
             ->sendOutputTo(storage_path('logs/import-mobile-last-run.log'));
 
+        //
+        // Shop
         $schedule->command('import:products-full', ['--yes'])
             ->dailyAt('23:10')
             ->withoutOverlapping(self::FOR_ONE_YEAR)
             ->sendOutputTo(storage_path('logs/import-products-full-last-run.log'));
 
-        $schedule->command('import:web-full', ['--yes'])
+        //
+        // Website
+        $schedule->command('import:web-full', ['articles', '--yes'])
             ->dailyAt('23:15')
             ->withoutOverlapping(self::FOR_ONE_YEAR)
-            ->sendOutputTo(storage_path('logs/import-web-full-last-run.log'));
+            ->sendOutputTo(storage_path('logs/import-web-full-articles-last-run.log'));
```

Comment:
Could we shorten the log file names to `logs/import-articles-last-run.log`?

Reply:
It is pretty verbose, but this is what the directory of log files currently looks like:

```
data-dump-last-run.log
delete-assets-last-run.log
delete-collections-last-run.log
import-assets-last-run.log
import-collections-last-run.log
import-enhancer-last-run.log
import-events-ticketed-last-run.log
import-mobile-last-run.log
import-products-full-last-run.log
import-queues-last-run.log
import-web-full-last-run.log
import-web-last-run.log
scout-import-agents-last-run.log
```

So with multiple `import-web-full-* processes running I wanted to be able to see those together.